src/Tools/ExternalAccess/LiveShare/Classification/RoslynClassificationService.Exports.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Composition;
+using Microsoft.CodeAnalysis.Classification;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.LiveShare.Classification
+{
+    [ExportLanguageServiceFactory(typeof(IClassificationService), StringConstants.CSharpLspLanguageName), Shared]
```

Comment:
do we need one for typescript until they move out?

Reply:
and work item to move those to typescript external access dll once they create one.

Reply:
Nope, there's only a couple places where the roslyn client is exporting typescript for clients newer than 16.0

https://github.com/dotnet/roslyn/projects/45#card-21807510